grabbit.sh

```diff
@@ -30,7 +30,7 @@ function newGrabbitRequest() {
     echo $SET_YELLOW
     echo "Processing....."
     echo $SET_NO_COLOR
-    curl -s -f -X PUT --data-binary "@$configpath" -u $username:$password -D /tmp/grabbit_headers $client$GRABBIT_JOB > /tmp/grabbit
+    curl -s -f -X PUT --header "Content-Type: text/plain" --data-binary "@$configpath" -u $username:$password -D /tmp/grabbit_headers $client$GRABBIT_JOB > /tmp/grabbit
```

Comment:
We may need to do something similar for https://github.com/TWCable/grabbit-cli/blob/master/src/main/java/com/twcable/grabbit/tools/jobstarter/JobStarter.java#L148 for 6.2 support